en/docs/api-gateway/next/quick-start-guide.md

```
@@ -106,6 +106,15 @@ curl -i http://localhost:8080/reading-list/v1.0/books
 curl -ik https://localhost:8443/reading-list/v1.0/books
 ```
 
+!!! note "Running on Windows"
+    The commands above assume a Linux/macOS shell. On Windows, run the one-time setup with the PowerShell script instead — it takes the same flags and provisions the same files:
+
+    ```powershell
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)